src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -309,9 +309,16 @@ public void selectNotification(By selectNotificationLocator) {
     }
 
     public void putAppInBackground(int duration) {
-        throw new NotImplementedException("To be migrated to appium 2.0");
+     //   throw new NotImplementedException("To be migrated to appium 2.0");
         // todo - implement for appium2.0
 //        ((AppiumDriver) driver).runAppInBackground(Duration.ofSeconds(duration));
+        String platform = String.valueOf(Runner.getPlatform());
+        if (platform.equals("android")) {
+            ((AndroidDriver) driver).runAppInBackground(Duration.ofSeconds(duration));
```

Comment:
Is it possible to make this generic (use AppiumDriver instead of AndroidDriver)?
If no then no problem, but if this can be done, make it generic

Reply:
app in background functionality is only present in android right now, in IOS it can be done through gesture now

so it is only implemented for Android